llvm/test/Transforms/InstCombine/icmp-fold-with-cast.ll

```diff
@@ -0,0 +1,185 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --tool ./bin/opt --version 3
+; See PRXXX for more details
+; RUN-./bin/opt: opt < %s -S -passes=ipsccp | FileCheck %s
```

Comment:
This is an ipsccp test in the InstCombine directory?

Reply:
yep, a typo on my end